### ISSUE TITLE
Announce Caprice32 V4.6.0 and news on sourceforge board.

### ISSUE THREAD

**Issue by zackizacki** [ColinPitrat/caprice32#156]
It's a pitty that the caprice32 project is recognized as dead on sourceforge. And they still download the 15 year old version (4.2.0) 29 times per week.
Announcing new versions and activities there on the discussion board could maybe invite more people for participation. Maybe people would have more time now during the current "public program" so maybe there could be more contributions and feedback as well?

**Comment by zackizacki** [comment]
Do we have some more detailed release notes with last changes for 4.6.0?

**Comment by ColinPitrat** [comment]
I updated the release notes:
- Support CAP32_DELAY in autorun commands
- Support German keyboard (physical layout only)
- Better freedesktop integration
- Better logic for finding configuration file
- Dynamic loading of CAPS (IPF) library
- Nix packaging support

**Comment by ColinPitrat** [comment]
Note that I already posted something 3 years ago:
![image](https://user-images.githubusercontent.com/1541863/76952920-c0493e80-6905-11ea-87e6-82b3c6bbc5df.png)

**Comment by zackizacki** [comment]
Yes, I've seen it, very good!
But 3 years is a long time and there is no indications in the message that there are new release ready for download on github.

Did you try to contact the orinial author?
Maybe finally he is happy or at least ok with the fork and he will support/promote it on "his site" also more officially?

**Comment by ColinPitrat** [comment]
I contacted the original author and he supports the fork but was unable to update sourceforge (I suppose because he lost his credentials).

**Comment by zackizacki** [comment]
What?? Really? :-)
This is a solvable little problem maybe via sourceforge service.
Here it is:
https://sourceforge.net/auth/recovery_support
I'm trying to find his email address to contact him, 
maybe we can get his account recovered so that some updates maybe with the release notes 
can be done there.

**Comment by ColinPitrat** [comment]
I'm not sure it's the reason, that's what I supposes as the account was more than 10 years old already.
I didn't insist. I'll ask him again.

**Comment by ColinPitrat** [comment]
Ulrich added a message in the project summary:
https://screenshot.googleplex.com/wU0Dh1g8n3U

**Comment by zackizacki** [comment]
That is really cool and very generous from ulrich to have that statement standing there! I'll write him a thank you mail. 

So, now it is more legitimate and still also useful to announce new releases with release notes maybe on the discussion board there? What do you think? I think investments in release announcements do pay off as the project does live from public attention at least to a certain degree.

**Comment by ColinPitrat** [comment]
Yes, that's an excellent idea. I announced some release (I don't remember which one) in a few places but I don't do it regularly. I'm a bit too lazy to do it but if you want to post some announces somewhere, you're more than welcome!
